### DISCUSSION TITLE
Trump and Project 2025

### DISCUSSION THREAD

**Discussion by aloh86** [weather-gov/api#773] [General]
So, assuming Project 2025 gets implemented now that Trump is elected, NWS will be dismantled. Where do we get weather data now? Private companies that charge $$$/month? If privatized and commercialized, how much are we looking to pay?

https://www.pbs.org/newshour/amp/politics/fact-checking-what-project-2025-says-about-the-national-weather-service-and-noaa

**Comment by wilzhub** [comment]
Trump doesnt care about project 2025, its just Qanon for lefties.

**Reply by aloh86** [reply]
Not sure which reality you live in @wilzhub.
https://www.msn.com/en-us/news/politics/here-are-the-5-project-2025-authors-trump-has-already-nominated-for-his-cabinet/ar-AA1uB6QP

**Reply by aloh86** [reply]
This aged well...

**Reply by breadbored** [reply]
He's implemented [42% of the Project 2025 agenda](https://www.project2025.observer/)

**Comment by Orcazephyr** [comment]
I am highly skeptical that NOAA and the NWS would be completely dismantled... Many Republican senators and representatives would be in favor of preserving at least the operational part of the NWS. Looking for long-term climate data might be problematic, but for forecasts I would think that the NWS API remains intact.

**Reply by aloh86** [reply]
Update: https://www.theguardian.com/us-news/2025/feb/04/doge-noaa-headquarters

**Reply by Orcazephyr** [reply]
Yeah it's not looking good. Nothing is set in stone but I am working on a contingency plan for my weather app. I plan to use ecmwf and gem data in the event that NOAA undergoes temporary or permanent structural changes

**Reply by Orcazephyr** [reply]
If the NBM can be replicated without the GFS component, that might be the closest we can get to a NOAA forecast.

**Reply by Orcazephyr** [reply]
I still don't expect any **immediate** impacts to operational forecasting as it is very important to aviation and the entire private weather sector. If you rely on any climate data, that is an entirely different story. 

I'm not saying it won't happen, but I would think that NOAA is low on the list of Project 2025's priorities.

**Comment by dangelsaurus** [comment]
> So, assuming Project 2025 gets implemented now that Trump is elected, NWS will be dismantled. Where do we get weather data now? Private companies that charge $$$/month? If privatized and commercialized, how much are we looking to pay?
> 
> https://www.pbs.org/newshour/amp/politics/fact-checking-what-project-2025-says-about-the-national-weather-service-and-noaa

I don't mind paying a little, but my issue with other services is with licensing. Darksky had very generous licensing, but many services don't allow storing (except for brief caching) data, or are CC-BY-SA (which is always questionable for me, when mixing in my own data.)

**Reply by JJKraw** [reply]
Remember that a whole lot of the data the private companies get _is_ from the NWS. If that's unavailable, it really doesn't matter if there's a fee or not. The information will be very incomplete.

**Reply by wsamoht** [reply]
I am surprised how many people don't realize this. I've heard people say, I have the weather app on my phone and get alerts with it, why do we need the NWS? Um, because most of that data comes from or is derived from the NWS!!!!! It would be a huge investment (if not impossible) for every private company to have their own weather balloons, buoy/water sensors, planes, etc.

**Reply by joepperkins** [reply]
👍

**Comment by StephenClouse** [comment]
As you can probably guess, this is not really the discussion we want to have here.

Having said that, the Office of Dissemination portfolio and budget with regards to the API have not been meaningfully affected by any recent organizational changes. API remains a part of NWS's long-term plans and we intend to continue to support and enhance it for the foreseeable future.
